Vagrantfile

```diff
@@ -118,7 +118,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
             end
 
             # mount the host directories
-            node.vm.synced_folder "../../../../", gopath_folder
+            node.vm.synced_folder "bin", File.join(gopath_folder, "bin")
+            node.vm.synced_folder ".", File.join(gopath_folder, "src/github.com/contiv/netplugin")
```

Comment:
@erikh most of us work on multiple Repos like deploy/libcompose/netplugin/objmodel/ofnet in same workspace.
So, we'd need entire source tree available inside the VM for builds.
Can we mount synced_folder $GOPATH/src as /opt/gopath/src inside VM?
this way, binaries don't conflict between Mac and Linux and we can still have entire source tree inside the VM